rpcs3/Emu/RSX/RSXThread.cpp

```diff
@@ -2379,13 +2383,14 @@ namespace rsx
 
 				for (u32 ea = address >> 20, end = ea + (size >> 20); ea < end; ea++)
 				{
-					u32 io = RSXIOMem.io[ea];
+					u32 io = iomap_table.io[ea];
 
-					if (io < 512)
+					if (io + 1)
 					{
+						io >>= 20;
 						unmap_status[io / 64] |= 1ull << (io & 63);
-						RSXIOMem.ea[io].raw() = 0xFFFF;
-						RSXIOMem.io[ea].raw() = 0xFFFF;
+						iomap_table.ea[io] = -1;
+						iomap_table.io[ea] = -1;
```

Comment:
Why change from hex to -1 here, but then leave the hex below? This negative one thing has come up quite a bit over the years, someone needs to decide on an official style. I suppose whatever you do, at least make it consistent. Great pr btw, cleans up the code well.

Reply:
0xffff comes from HLE gcm writing into a u16, writing -1 just causes a warning on truncation there.
Here -1 is written to u32, there is no such problem here.